cmd/hyperkube/hyperkube_test.go

```diff
@@ -177,15 +177,15 @@ func TestServerHelp(t *testing.T) {
 	x := runFull(t, "hyperkube test1 --help")
 	assert.NoError(t, x.err)
 	assert.Contains(t, x.output, "A simple server named test1")
-	assert.Contains(t, x.output, "--help[=false]: help for hyperkube")
+	assert.Contains(t, x.output, "-h, --help                                               help for hyperkube")
 	assert.NotContains(t, x.output, "test1 Run")
 }
 
 func TestServerFlagsBad(t *testing.T) {
 	x := runFull(t, "hyperkube test1 --bad-flag")
 	assert.EqualError(t, x.err, "unknown flag: --bad-flag")
 	assert.Contains(t, x.output, "A simple server named test1")
-	assert.Contains(t, x.output, "--help[=false]: help for hyperkube")
+	assert.Contains(t, x.output, "-h, --help                                               help for hyperkube")
 	assert.NotContains(t, x.output, "test1 Run")
```

Comment:
This looks like a little bit strange, the help output really contains so many extra spaces?